spring-integration-gcp/src/main/java/org/springframework/integration/gcp/outbound/PubSubMessageHandler.java

```diff
@@ -29,15 +37,28 @@
 
 	private final PubSubOperations pubSubTemplate;
 
+	private MessageConverter messageConverter;
```

Comment:
I think we would want the message converter to be configurable from the outside.

Reply:
👍

Reply:
But we don't use message converter for anything here.
What do you want to use it for?

Reply:
??? Really? Why have you added it here then at all?
I see below the code like:
```
this.messageConverter.fromMessage()
```
So, looks like you use it here.
Gotcha?

The idea is to let end user to inject externally configured `MessageConverter`. We need setter for this property. That's all.

Reply:
Woops, I'm sorry, I thought this was `PubSubTemplate`.
Yes of course, we totally need to allow users to specify a converter!

Reply:
Added a message conversion setter.